src/derive.rs

```diff
@@ -182,16 +192,58 @@ impl<'a> DerivedModule<'a> {
             new_h
         })
     }
-    // remap a const expression from source context into our derived context
-    pub fn import_const_expression(&mut self, h_cexpr: Handle<Expression>) -> Handle<Expression> {
-        self.import_expression(
-            h_cexpr,
-            &self.shader.as_ref().unwrap().const_expressions,
-            self.const_expression_map.clone(),
-            self.const_expressions.clone(),
-            false,
-            true,
-        )
+
+    // remap either a const or pipeline override expression from source context into our derived context
+    pub fn import_global_expression(
+        &mut self,
+        h_expr: Option<Handle<Expression>>,
+    ) -> Option<Handle<Expression>> {
```

Comment:
could we make this Handle -> Handle instead of Option<Handle> -> Option<Handle> and use `pipeline_override.init.map(|init| self.import_global_expression(init))` in `import_pipeline_override` to avoid the unwraps?